distribution/pom.xml

```diff
@@ -197,6 +197,8 @@
                                         <argument>-c</argument>
                                         <argument>org.apache.druid.extensions:druid-avro-extensions</argument>
                                         <argument>-c</argument>
+                                        <argument>org.apache.druid.extensions:druid-azure-extensions</argument>
```

Comment:
You should also remove this from the [`bundle-contrib-exts` profile](https://github.com/apache/druid/blob/master/distribution/pom.xml#L385)

Reply:
fixed